### ISSUE TITLE
Using the create function manually 

### ISSUE THREAD

**Issue by nishantwrp** [typescript-eslint/typescript-eslint#2387]
**Repro**

```javascript
const { rules: { 'no-explicit-any': rule } } = require('@typescript-eslint/eslint-plugin');

module.exports = {
  rules: {
    'test-rule': {
      ...rule,
      create: function(context) {
        ... my code
        if (some-condition) {
          return rule.create(context);
        }
      }
    }
  }
};

```
**Expected Result**
The eslint check should work when the condition is true.

**Actual Result**

It doesn't

**Additional Info**
Please tell me if I'm doing something wrong

**Comment by bradzacher** [comment]
Could you please explain more about what you're trying to do here?

**Comment by nishantwrp** [comment]
Actually i want to trigger the check when I detect a fixed line in the file (using the getSource eslint API).

**Comment by bradzacher** [comment]
Again - could you clarify what are you trying to do?

**Comment by nishantwrp** [comment]
I am creating a custom-eslint-rule that uses `no-explicit-any` rule of this plugin and triggers the rule only when if detects a line in the code. Is it clear, now?

**Comment by bradzacher** [comment]
There is nothing obviously wrong with the snippet of code you've provided.
Have you tried using a debugger to inspect if your logic to ensure it's working how you expect?

**Comment by nishantwrp** [comment]
I've tried that and came to know that `rule.create(context);` return `{}`. Any ideas?

**Comment by bradzacher** [comment]
Nope - it's working fine for me when I try it out locally.
Likely there's something wrong with the code you didn't send?

![image](https://user-images.githubusercontent.com/7462525/89958263-4d12d280-dbee-11ea-81d4-873a62b66026.png)
